docs/metabase/setup.md

```diff
@@ -0,0 +1,64 @@
+# Setting up Metabase on AWS
+
+The instructions on [Metabase docs](https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-elastic-beanstalk.html) are already quite good. This document is just a few salient points to assist the docs to fit with our infrastructure. The docs use Elastic Beanstalk, which runs Metabase in a docker container managed by AWS.
+
+## Caveat Lector
+
+* The entire Elastic Beanstalk UI is an SPA, so don't forget to save your changes before you launch your app in the intermediary steps and/or press the back button because you will have to start off the whole process again.
```

Comment:
The first clause about saving changes is something we _should_ do, but the second clause about hitting "back" is something we _shouldn't_ do? The sentence structure is a bit confusing. Could you perhaps split it up?

`...don't forget to save your changes before you launch your app in the intermediary steps. Do not press the back button during setup either, as you will have to start off the whole process again.`

Reply:
Ah, yeah, that's better. Apologies for my smarminess.

Reply:
Hehe that's what review's for 😅 No need for apologies.